Makefile

```diff
@@ -142,7 +142,7 @@ test-unit-coverage: test-unit
 	gocov report build/coverage/coverprofile.json
 
 .PHONY: test-e2e
-test-e2e: crds test-e2e-plain
+test-e2e: install-strategies test-e2e-plain
```

Comment:
nice! I tested this, we are all good! I hope that in the future we can use the `make install-operator` to deploy the operator, instead of using the operator-sdk. Of course, for this we would need a different image, we cannot use `latest`, but an image with the code of the current branch.